BrianLusina/hangman
# Game logic & Game Loop

## Description

Game Logic and game loop to get the game running until the end

## How can we test this?

- `make install` to install dependencies
- `make lint` to run linting
- `make test` to run tests
- `make run` to run the game
- The game should start in the terminal and you can guess some letters :)

## Checklist:
